WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -568,8 +568,25 @@ - (void)showOnboardingTips
 
 #pragma mark - Actions
 
-- (void)showBlogSelectorPrompt
+/**
+ *	@brief      Handles the UIControlEventTouchUpInside event for the blog selector button.
+ *	@details    This method handles a the touch up inside event for the blog selector button.
+                Since there are two different modes this button can have, we have a few
+                different scenarios to consider:  1) If the user is editing an existing post 
+                exit the screen we are currently on and return. 2) If the user is creating a
+                new post and the post does not have site-specific changes, show the blog selector
+                3) If the user is creating a new post and the post does have site-specific 
+                changes, display a blog change warning.
+ *
+ *	@param      sender The WPBlogSelectorButton triggering this action.
+ */
```

Comment:
Well, we can consider this function to be properly documented for sure! hah :+1: